subsys/bluetooth/mesh/vnd/silvair_enocean_srv.c

```diff
@@ -14,6 +14,12 @@
 #define BT_MESH_SILVAIR_ENOCEAN_PROXY_SUB_OP_DELETE 0x02
 #define BT_MESH_SILVAIR_ENOCEAN_PROXY_SUB_OP_STATUS 0x03
 
+#define PHASE_B_DELAY 200
```

Comment:
Minor. Isn't it 150ms?
<img width="354" alt="Screenshot 2022-01-28 at 15 26 37" src="https://user-images.githubusercontent.com/677018/151563998-3c3c64ef-a48c-4a76-9f98-39a1c6a8d957.png">

Reply:
hmm... yes indeed